meerkat_analysis/indicators.py

```diff
@@ -60,20 +70,32 @@ def count(data, var_id, start_date=None, end_date=None, epi_week_start_day=None)
                                            epi_week_start_day)
 
     dates = pd.date_range(start_date, end_date, freq=freq, closed="left")
-
+    return_val=0.0
+    if calc==ot:
```

Comment:
`if calc==ot:` Does this work? You don't seem to have defined the `ot` variable anywhere. 
Shouldn't it be `if calc is "ot"`?

In fact if you were super keen you could use a dictionary based model:
```
calculation = {
    "ot": pd[var_id].sum(),
    "om": pd[var_id].mean(),
    "omin": pd[var_id].min()
}
[...]
return calcuation[calc], timeline
```